modules/ROOT/pages/access-logging.adoc

```diff
@@ -48,15 +48,15 @@ Use the `accessLogFormat` property to specify the information  and format you wa
 This log format string is specified by using the `logFormat` attribute of `httpAcccessLogging` or `accessLogging` elements in the `server.xml` file, as shown in the following examples:
 
 ----
-<httpAccessLogging logFormat='%h %u %{t}W "%r" %s %b'/>
+<httpAccessLogging logFormat='%h %u %{t}W "%r" %s %b %D %{R}W'/>
```

Comment:
To clarify- is `%{R}W` an actual attribute  value or does `{R}` represent a variable that would change depending on the config details

Reply:
`%{R}W` is [the actual attribute value](https://openliberty.io/docs/latest/access-logging.html#_http_access_log_format):

> %{R}W | Service  time of the request from the moment the request is received until the  first set of bytes of the response is sent - millisecond accuracy,  microsecond precision